Sources/SuperDelegate+UserNotifications.swift

```diff
@@ -24,6 +24,7 @@ import Foundation
 // MARK: UserNotificationCapable – Opting into this protocol gives your app the ability to register for User Notifications. Adopt RemoteNotification or LocalNotification to actually receive notifications.
 
 
+@available(iOS, deprecated: 10.0)
```

Comment:
All this deprecations is going to do is propagate warnings down into wherever apps adopt this protocol, right?

Given that we have no alternative (i.e. that #3 is outstanding), I strongly suggest that we don't deprecate this category yet.